README.md

```diff
@@ -268,6 +268,10 @@ metadata:
     yawol.stackit.cloud/logForward: "true"
     # Defines loki URL for the log forwarding.
     yawol.stackit.cloud/logForwardLokiURL: "http://example.com:3100/loki/api/v1/push"
+    # Defines labels that are added when forwarding logs
+    # The prefix "logging.yawol.stackit.cloud/" will be trimmed
+    # and only "foo": "bar" will be added as a label
+    logging.yawol.stackit.cloud/foo: "bar"
```

Comment:
I would add a second label to make clear that it is possible to set multible ones.